ports/mimxrt/Makefile

```diff
@@ -165,6 +169,8 @@ SRC_C += \
 	shared/runtime/stdout_helpers.c \
 	shared/runtime/sys_stdio_mphal.c \
 	shared/timeutils/timeutils.c \
+	lib/oofatfs/ff.c \
+	lib/oofatfs/ffunicode.c \
```

Comment:
these additions are not needed, they are already included in `SRC_MOD` via `extmod/extmod.mk`

Reply:
fixed.